### PR TITLE
Document ES2015 compilation.

### PR DESCRIPTION
**Pull request by cycomachead**
There might be a more appropriate place or more thorough explanation, but this seemed like a start.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,8 +120,9 @@ npm install autotrack
 // In your JavaScript code
 import 'autotrack';
 ```
+_Note autotrack's source is published as ES2015, and you will need to make sure you're not excluding it from compilation. See #137 for more details._
```
Comment (philipwalton) [inline README.md]: Can you format this to be the same as other notes in the README? Where the word "note" is in bold. Also please link to the issue.

File: README.md [hunk 2]
```diff
@@ -120,8 +120,9 @@ npm install autotrack
...
 
-The above `import` statement will include all autotrack plugins in your generated source file. If you only want to include a specific set of plugins, you can import them individually:
+1The above `import` statement will include all autotrack plugins in your generated source file. If you only want to include a specific set of plugins, you can import them individually:
```
Comment (philipwalton) [inline README.md]: Can you remove the "1" at the beginning of this line?

Comment (cycomachead) [inline README.md]: Wow, that was mighty sloppy of me. :/

**Comment by cycomachead** [comment]
Done. :)

**Comment by philipwalton** [comment]
Thanks!

**Pull request merged**